src/components/calculator.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable react/button-has-type */
+/* eslint-disable no-console */
+import React, { useState } from 'react';
+import '../index.css';
+
+function Calculator() {
```

Comment:
- [ ] The `Calculator.js` component should be class-based so as to meet the project requirement.